### ISSUE TITLE
Don't download node+npm if it's already installed in the OS

### ISSUE THREAD

**Issue by mvysny** [vaadin/vaadin-gradle-plugin#11]
**Desktop (please complete the following information):**
 - OS: Ubuntu 19.10 x86-64 with nodejs and npm installed via apt
 - JDK version: 8
 - Gradle version: 5.6.4
 - Vaadin Plugin version: 0.1.0

**Describe the bug**
The gradle plugin tries to download nodejs and npm even though I have them installed via `apt install npm`

**To Reproduce**

1. Make sure that npm is installed: sudo apt install npm
2. git clone git clone https://github.com/mvysny/karibu10-helloworld-application && cd karibu10-helloworld-application && git checkout vaadin14-plugin 
3. ./gradlew clean build

**Expected behavior**
The system npm is used

**Actual**
node and npm is attempted to be downloaded, which fails when offline:

```
Execution failed for task ':nodeSetup'.
> Could not resolve all files for configuration ':detachedConfiguration3'.
   > Could not download node-linux-x64.tar.gz (org.nodejs:node:11.15.0)
      > Could not get resource 'https://nodejs.org/dist/v11.15.0/node-v11.15.0-linux-x64.tar.gz'.
         > Could not GET 'https://nodejs.org/dist/v11.15.0/node-v11.15.0-linux-x64.tar.gz'.
            > nodejs.org: Temporary failure in name resolution

```

**Comment by mvysny** [comment]
Solved when #18 is implemented.

**Comment by mvysny** [comment]
The npm/node will now never be downloaded - the build will fail if those tools are not installed locally. I will add examples to the example projects on how to use a specialized Gradle plugin to download those locally.

Fixed in 401b0e02b56e42288aff1ed4adbaf2b4f2ea4d8c
